src/s2iExec.ts

```diff
@@ -7,35 +7,42 @@ import * as fs from "fs";
 import * as path from "path";
 import { Command } from "./command";
 import { Installer } from "./installer";
-import { Inputs } from "./generated/inputs-outputs";
+import { Inputs, Outputs } from "./generated/inputs-outputs";
 import {
     BinaryVersion, convertStringToBinaryVersion, FindBinaryStatus, getReason,
 } from "./utils/execHelper";
 
 export async function run(): Promise<void> {
+    const DEFAULT_TAG = "latest";
     const builderImage = core.getInput(Inputs.BUILDER_IMAGE, { required: true });
-    const imageName = core.getInput(Inputs.IMAGE_NAME, { required: true });
-    const imageTag = core.getInput(Inputs.IMAGE_TAG, { required: false });
+    const image = core.getInput(Inputs.IMAGE, { required: true });
+    const tags = core.getInput(Inputs.TAGS, { required: false });
     const pathContext = core.getInput(Inputs.PATH_CONTEXT, { required: false });
     const logLevel = core.getInput(Inputs.LOG_LEVEL, { required: false });
     const envVars = core.getInput(Inputs.ENV_VARS, { required: false });
     const runnerOS = process.env.RUNNER_OS || process.platform;
 
-    const version = "v1.3.1";
-    core.debug(version);
-    core.debug(runnerOS);
+    const tagsList: string[] = tags.split(" ");
 
+    const version = "v1.3.1";
     const binaryVersion: BinaryVersion = convertStringToBinaryVersion(version);
     const s2iBinary: FindBinaryStatus = await Installer.installS2i(binaryVersion, runnerOS);
 
     if (s2iBinary.found === false) {
         throw new Error(getReason(s2iBinary));
     }
-
     Installer.addS2iToPath(s2iBinary.path, runnerOS);
-    await Command.execute(s2iBinary.path, [ "version" ]);
 
-    const buildCmd = [ "build", pathContext, builderImage, `${imageName}:${imageTag}`,
+    core.debug(version);
+    core.debug(runnerOS);
+
+    // info message if user doesn't provides any tag
+    if (!tagsList.length) {
+        core.info(`Input "${Inputs.TAGS}" is not provided, using default tag "${DEFAULT_TAG}"`);
+        tagsList.push(DEFAULT_TAG);
+    }
+
+    const buildCmd = [ "build", pathContext, builderImage, `${image}:${tagsList[0]}`,
```

Comment:
remember that multi-line array syntax should be like this

```js
const a = [
  "one", "two"
];
```

or, if there are more than a few items:
```js
const a = [
  "one", 
  "two",
  "three",
  "four",
];
```